release_docs/RELEASE.txt

```diff
@@ -1439,7 +1439,11 @@ Bug Fixes since HDF5-1.12.0 release
 
     High-Level Library
     ------------------
-    -
+    - Fixed HL_test_packet, test for packet table vlen of vlen.
+
+      Incorrect length assignment.
+
+        (ADB - 2021/10/14)
 
```

Comment:
If it was truly a typo, I think it can be said here.  When I read "Incorrerct length assignment", I thought why was it incorrect, how do I know now is correct.  But, just my 2c, and really, just 2c.

Reply:
Multiplication would provide a different length then addition - therefore incorrect assignment.

Reply:
vv is designated as a loop counter, and vv + 1 matches the length of the allocation in line 537.  i suppose it's for null termination.  Multiplying it by 1 as it was makes no sense, since it would have the same value as vv.

Reply:
Well the code was basically a copy of the same test in test - doing a comparison showed the problem - seemed like a typo.